docs/cloud-native-security/vuln-management-dashboard.asciidoc

```diff
@@ -0,0 +1,26 @@
+[[vuln-management-dashboard]]
+= Cloud Native Vulnerability Vulnerability Management Dashboard
+
+The Cloud Native Vulnerability Management (CNVM) dashboard gives you an overview of the vulnerabilities detected in your cloud infrastructure by the CNVM integration.
+
+NOTE: To learn how to collect this data, refer to <<vuln-management-get-started, Get Started with CNVM>>.
+
+image::images/vuln-management-dashboard.png[The CNVM dashboard]
+
+.Requirements
+[sidebar]
+--
+* The CNVM dashboard is available to all Elastic Cloud users. For on-prem deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].
+--
+
+[[CNVM-dashboard-UI]]
+== CNVM dashboard UI
+At the top of the dashboard, 5 summary cards appear. They display the number of monitored cloud accounts, scanned virtual machines, and vulnerabilities (grouped by severity). The *Trend by severity* bar graph complements the summary cards by displaying the number of vulnerabilities found on your infrastructure over time, sorted by severity. To view data from specific cloud accounts on this graph, select the account names from the *Accounts* drop down menu.
+
+The page also includes three tables:
+
+* *Top 10 vulnerable resources* shows your VMs with the highest number of vulnerabilities.
+* *Top 10 patchable vulnerabilities* shows the vulnerabilities that were found most often in your environment and can be fixed by a software update.
+* *Top 10 vulnerabilities* shows the most common vulnerabilities in your environment, with additional details.
+
+Each table includes a *View all vulnerabilities* button that opens the <<vuln-management-findings, Vulnerabilities Findings>> page, where you can view additional details.
```

Comment:
Super tiny nit here - I don't think the **View all vulnerabilities** link counts as button in the UI. 

```suggestion
At the bottom of each table, click *View all vulnerabilities* to open the <<vuln-management-findings, Vulnerabilities Findings>> page, where you can view additional details.
```